cycli/main.py

```diff
@@ -117,12 +120,57 @@ def run(self):
                 document = cli.run()
                 query = document.text
 
+                print(query)
```

Comment:
This should be removed. We don't want to print the query as it's already visible.